docs/examples/ansible/index.md

```diff
@@ -5,15 +5,10 @@ date: 2021-03-18 12:17:00 --0600
 categories:
 ---
 
-- [Ansible collections for](#overview)
```

Comment:
TOCs are now generated by mkdocs.  this eliminates the need to generate these internal to the document.